Library/Homebrew/extend/ENV/super.rb

```diff
@@ -67,6 +67,11 @@ def setup_build_environment(formula = nil)
     self["HOMEBREW_INCLUDE_PATHS"] = determine_include_paths
     self["HOMEBREW_LIBRARY_PATHS"] = determine_library_paths
 
+    if MacOS::Xcode.without_clt?
+      self["MACOSX_DEPLOYMENT_TARGET"] = MacOS.version.to_s
+      self["SDKROOT"] = MacOS.sdk_path
```

Comment:
Wasn't this originally `MacOS.sdk_path || "/"`? And does `MacOS::Xcode.without_clt?` really imply that Xcode is installed and a usable SDK path will be returned by `MacOS.sdk_path`?